examples-standalone/kendoangular-java-integration/readme.md

```diff
@@ -0,0 +1,93 @@
+# Kendo UI for Angular Grid and Upload Component with Java Spring Boot: A Sample Project
+
+This sample project demonstrates how to utilize Kendo UI for Angular's [Grid](https://www.telerik.com/kendo-angular-ui/components/grid/) and [Upload](https://www.telerik.com/kendo-angular-ui/components/uploads/upload/) components with a Java Spring Boot Maven backend.
+
+For more detailed information, refer to the [Java integration article](https://www.telerik.com/kendo-angular-ui/components/installation/java-spring).
+
+## Getting Started
+
+Follow these steps to set up and run the project locally:
+
+### 1. Install Required Tools
+
+Ensure you have the following tools installed on your machine:
+
+- **Angular CLI (18.0.0 or later)**: A command-line interface for Angular. [Download Angular CLI](https://angular.io/cli).
+- **Java Development Kit (JDK 23)**: Essential for running Java applications. [Download JDK](https://www.oracle.com/java/technologies/downloads/).
```

Comment:
I have no clue which one should be installed:
<img width="316" alt="image" src="https://github.com/user-attachments/assets/a6586b2e-c4e0-41cf-bcd9-0f32a895f2ba">

Pointing the user to the exact product that needs to be installed might feel better.

Reply:
I will add a note, but generally ARM chips are M1 > series and x64 are Intel based Macs.